sources/packages/backend/apps/db-migrations/src/sql/Applications/Drop-add-application-versioning-columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE
+  sims.applications DROP COLUMN parent_application_id,
+  DROP COLUMN preceding_application_id;
```

Comment:
Even though postgres drops the constraints silently, I would suggest the rollback dropping the constraint mindfully.